docs/contracts/oracle-report-sanity-checker.md

```diff
@@ -3,6 +3,436 @@
+[AccessControlEnumerable](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/utils/access/AccessControlEnumerable.sol)
+contract and a bunch of [granular roles](#permissions).
+
+## Limits List
+
+`OracleReportSanityChecker` introduces a new type `LimitsList` which contains all the limits used by the contract.
+Below is a detailed description of the limits contained in the `LimitsList` struct:
+
+- **`uint256 churnValidatorsPerDayLimit` ∈ [0, 65535]** - the max possible number of validators that might been reported as _**appeared**_
+  or _**exited**_ during a single day. [`AccountingOracle`](./accounting-oracle.md) reports validators as _**appeared**_ once them become
+  _**pending**_ (might be not _**activated**_ yet). Thus, this limit should be high enough for such cases because Consensus Layer has no
+  intrinsic churn limit for the amount of _**pending**_ validators (only for _**activated**_ instead).
+  For Lido it's limited by the max daily deposits via [`DepositSecurityModule`](./deposit-security-module.md). In contrast, _**exited**_ are reported according to the
+  [Consensus Layer churn limit](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_validator_churn_limit).
+- **`uint256 oneOffCLBalanceDecreaseBPLimit` ∈ [0, 10000]** - the max decrease of the total validators' balances on the Consensus Layer since
+  the previous oracle report. Represented in the Basis Points (100% == 10000).
+- **`uint256 annualBalanceIncreaseBPLimit` ∈ [0, 10000]** - the max annual increase of the total validators' balances on the Consensus Layer
+  since the previous oracle report. Represented in the Basis Points (100% == 10000).
+- **`uint256 simulatedShareRateDeviationBPLimit` ∈ [0, 10000]** - the max deviation of the provided `simulatedShareRate` and the actual one within the
+  currently processing oracle report. Represented in the Basis Points (100% == 10000).
+- **`uint256 maxValidatorExitRequestsPerReport` ∈ [0, 65535]** - the max number of exit requests allowed in report
+  to [ValidatorsExitBusOracle](./validators-exit-bus-oracle.md)
+- **`uint256 maxAccountingExtraDataListItemsCount` ∈ [0, 65535]** - the max number of data list items reported to accounting oracle in extra data
+- **`uint256 maxNodeOperatorsPerExtraDataItemCount` ∈ [0, 65535]** - the max number of node operators reported per extra data list item
+- **`uint256 requestTimestampMargin` ∈ [0, type(uint64).max]** - the min time required to be passed from the creation of the request to be finalized till the time of the oracle report
+- **`uint256 maxPositiveTokenRebase` ∈ [1, type(uint64).max]** - the max positive token rebase allowed per single oracle report token rebase
+  happens on total supply adjustment, huge positive rebase can incur oracle report sandwiching.
+  Uses 1e9 precision, e.g.: 1e6 - 0.1%; 1e9 - 100%; type(uint64).max - unlimited rebase.
+
+## Sanity Checks
+
+### `smoothenTokenRebase()`
```

Comment:
Looks like it's not really a sanity check, because it just calculate the sane amounts and does not validate anything. Can we name it differently and say a word about in Preface?

Reply:
Agreed, moved it into the "View Methods" section, cause it's actually a getter.